src/bulkActionPopup.js

```diff
@@ -122,17 +123,16 @@ export default function bulkActionPopupFactory(config) {
 
         // renders the component
         .on('render', function() {
-            var self = this;
-            var $element = this.getElement();
-            var $reason;
+            const self = this;
```

Comment:
While moving to ES6, it could be good too to also get rid of lexical scope enforcer (using arrow functions instead when the lexical scope should be the parent one)
Also using shorthand notation for object members: `validate: function validate() { ... }`  => `validate() { ... }`.
Those changes are not heavy and won't change the behavior, so we should apply them when we touch a file and it is not too big.

Reply:
changed to arrow functions and shorthand notation